config.py

```diff
@@ -0,0 +1,25 @@
+"""Flask configuration."""
+from os import environ, path
+from dotenv import load_dotenv
+
+basedir = path.abspath(path.dirname(__file__))
+load_dotenv(path.join(basedir, '.env'))
+
+
+class Config:
```

Comment:
`Config` should define all available configurations, while its subclasses (`ProdConfig`, `DevConfig`) just specify all or some of these configuration fields. if I am given a `Config,` I should not care if it is `DevConfig` or `ProdConfig`. i.e. `Config` subclasses do not extend the set of config fields.

Take a look: https://flask.palletsprojects.com/en/0.12.x/config/#development-production
`DEBUG`, `TESTING` and `DATABASE_URI` are all defined on `Config` level, while `ProductionConfig` and `DevelopmentConfig` just specify some values.

Reply:
fixed

Reply:
check: `PATH_FOR_INITIAL_DATA` is not defined in `Config`

Reply:
hey, please, be attentive to the code and comments: all configuration fields must be present, "announced" on `Config` level first of all, on `{Env}Config` level you just provide an exact value for the previously announced field.
If you use `PATH_FOR_INITIAL_DATA` then it should be somehow announced at `Config`

Reply:
Understood